src/bitmessagekivy/baseclass/draft_widgets.py

```diff
@@ -0,0 +1,26 @@
+# pylint: disable=no-member, too-many-arguments, too-few-public-methods
+
+"""
+Draft screen widgets are here.
+"""
+
+from kivymd.uix.label import MDLabel
+
+
+class HelperDraft(object):
```

Comment:
this can probably made more general into a message list mixin. I don't see anything draft-specific here